components/twitter/user-tweets.js

```diff
@@ -27,37 +25,29 @@ module.exports = {
   }, 
   methods: {},
   async run(event) {
-    const from = `from:${this.from.replace('@','')}`
+    const screen_name = this.screen_name //.replace('@','')
     const since_id = this.db.get("since_id")
-    const { lang, locale, geocode, result_type, enrichTweets, includeReplies, includeRetweets, maxRequests, count } = this
-    let q = from, max_id, limitFirstPage
-    
-    // join "from" filter and search keywords
-    if (this.q) {
-      q += ` ${this.q}`
-    }
-    
-    if (!since_id) {
-      limitFirstPage = true
+    const { enrichTweets, includeReplies, includeRetweets, count } = this
+    let max_id
+
+    if(!includeReplies) {
```

Comment:
Since you just pass `exclude_replies` directly to the `getUserTimeline` method, can you remove this and just pass:

```javascript
exclude_replies: !includeReplies
```

directly, instead?

Reply:
done!